marimo/_ast/app.py

```diff
@@ -398,6 +399,29 @@ async def embed(self) -> AppEmbedResult:
             )
 
 
+class AppMeta:
+    """
+    Metadata about the app.
+
+    This is used to store metadata about the app
+    that is not part of the app's code or state.
+    """
+
+    def __init__(self) -> None:
+        self.user_config = load_config()
+        self.apply_theme()
+
+    @property
+    def theme(self) -> str:
+        """The display theme of the app."""
+        return self.user_config["display"]["theme"] or "light"
+
+    def apply_theme(self) -> None:
+        """Apply the theme to the app."""
```

Comment:
can update this: 
"""
Apply the theme (light/dark) to third party libraries. 
If the theme is set to "system", then we fallback to "light".
"""